lib/contentTypeParser.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 const lru = require('tiny-lru')
+const parse = require('jitson')()
```

Comment:
I don't think we should have a shared instance of `jitson`, it should be unique.
@mcollina what do you think?

Reply:
By unique do you mean for each `ContentTypeParser` created?

Reply:
Yes! In the same way as we are doing with `tiny-lru` :)

Reply:
I thought a lot, and although I find it a bit ugly this is the best solution I could come up with.

`defaultJsonParser` doesn't like binding `this` so there was no way other then a closure 🤔